### PR TITLE
Translating 3.0/zh/core-libraries/xml.rst to chinese languge.

### PR DESCRIPTION
**Pull request by panxp**
Translating 3.0/zh/core-libraries/xml.rst to chinese languge.

### CODE REVIEW

**Comment by panxp** [comment]
Why The Travis CI build failed? Who can tell me.

**Comment by antograssiot** [comment]
@panxp I think that the ``meta`` tag should be the last thing in the file like in the en translation.

Try to move it and see if it does fix the build.

File: zh/core-utility-libraries/xml.rst [hunk 1]
```diff
@@ -1,18 +1,160 @@
...
+如果你的输入参数是无效的，Xml类会抛出一个异常::
+
+    $xmlString = 'What is XML?':
+    try {::
+        $xmlObject = Xml::build($xmlString); // Here will throw an exception
+    } catch (\Cake\Utility\Exception\XmlException $e) {
+        throw new InternalErrorException();
+    }
+
+.. 提示::
```
Comment (markstory) [inline zh/core-utility-libraries/xml.rst]: These directives have to stay in english. Sphinx cannot process translated commands.

**Comment by antograssiot** [comment]
@panxp I didn't read carefully the first time but this is supposed to be 3.x documentation and the PR has been open against the master branch (2.x).
Closing as we can't update the target branch.
Please reopen against 3.0 if not done already

**Pull request closed without merging**